### PR TITLE
Adding slack notification to hypershift cases for 4.13.

### PR DESCRIPTION
**Pull request by huangmingxia**
@heliubj18   @LiangquanLi930  @liangxia  Could you help review it? thanks.

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 1]
```diff
@@ -11007,6 +11007,16 @@ periodics:
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-ovn-hypershift-guest-p1-f21-destructive
+  reporter_config:
+    slack:
+      channel: '#forum-prow-hypershift-qe-ci'
```
Comment (LiangquanLi930) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:11012]: change to 
```
      job_states_to_report:
      - failure
      - error
```

Comment (huangmingxia) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:11012]: @LiangquanLi930 I think we should add success job, b/c some success jobs also has failed cases. For aborted and pending we can remove. WDYH?

Comment (LiangquanLi930) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:11012]: ack

Comment (LiangquanLi930) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:11012]:
```
      job_states_to_report:
      - failure
      - error
      - success
```

**Pull request merged**